### PR TITLE
Add custom error page for demo

### PR DESCRIPTION
**Pull request by marikrebega**
[Trello Task](https://trello.com/_____)

**CHECK LIST**
- [ ] specs added
- [x] specs passed
- [x] code coverage >90%
- [ ] DB indexes added if needed
- [ ] seed.rb file is updated if needed
- [x] PR is reviewed manually again (to make sure you have 100% ready code)
- [x] PR doesn't have conflicts with main branch
- [x] I've checked new feature as logged in and logged out user if needed
- [ ] "let!" is used in specs
- [x] screenshot of feature is added if task has UI

Знімок екрану з 2020-10-22 14-28-34

After clicking on the button
Знімок екрану з 2020-10-22 14-30-29

### CODE REVIEW

File: app/views/home/custom_error.html.slim [hunk 1]
```diff
@@ -0,0 +1 @@
+- 1/0
```
Comment (mmarusyk) [inline app/views/home/custom_error.html.slim:1]: no newline at end of file

**Review by mmarusyk — APPROVED** [review]

File: app/views/home/custom_error.html.slim (hunk 1 above)
Comment (Axedez) [inline app/views/home/custom_error.html.slim:1]: New line!

**Review by Axedez — APPROVED** [review]

File: app/views/home/custom_error.html.slim (hunk 1 above)
Comment (marikrebega) [reply app/views/home/custom_error.html.slim:1]: done

Comment (marikrebega) [reply app/views/home/custom_error.html.slim:1]: done

**Review by IvanDanyliv — APPROVED** [review]

**Review by chu4 — APPROVED** [review]

**Pull request merged**